website/source/docs/builders/azure-chroot.html.md.erb

```diff
@@ -0,0 +1,192 @@
+---
+description: |
+    The azure-chroot Packer builder is able to create Azure Managed Images leveraging
+    a VM in Azure.
+layout: docs
+page_title: 'Azure chroot - Builders'
+sidebar_current: 'docs-builders-azure-chroot'
+---
+
+# Azure Builder (chroot)
+
+Type: `azure-chroot`
+
+The `azure-chroot` builder is able to build Azure managed disk (MD) images. For
+more information on managed disks, see [Azure Managed Disks Overview](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview).
+
+The difference between this builder and the `azure-arm` builder is that this
+builder is able to build a managed disk image without launching an Azure VM 
```

Comment:
Just casually reading this, it seems to imply that you don't have to launch an instance at all, not that the instance is already launched. You do clarify it later, so it's probably fine, but maybe you could say "build a managed disk image using an already-launched Azure VM instance" instead?